plugins/chat/assets/javascripts/discourse/components/reviewable-refresh/chat-message.gjs

```diff
@@ -0,0 +1,7 @@
+import ReviewableChatMessage from "../reviewable-chat-message";
```

Comment:
The UI uses the existence of the appropriate component in the `reviewable-refresh` directory to determine if it should use the new UI. In this case, we don't really need any changes, so we can just use the old UI.